packages/storybook/README.md

```diff
@@ -0,0 +1,18 @@
+## Storybook
+
```

Comment:
I will try to prepare

Reply:
In my opinion the main goal fo storybook is to have an environment that can help easily show all components and they stages with different prop values. Also it can be used for tests and can use some special Vuex state to test the state as well. 
I will prepare the text a bit later.

Reply:
@barriebyron I copied intro from https://storybook.js.org website. Please, tell me is that enough? Maybe I need to add something?

Reply:
@barriebyron have a look at the last version. Hope that the purpose of storybook is more clear than it was.

Reply:
We'll improve upon this later on.